### PR TITLE
[FIX] Access denied to employee

### PR DESCRIPTION
**Pull request by bodedra**

### CODE REVIEW

**Comment by pedrobaeza** [comment]
I don't think this should be so opened. Please restrict a bit more. Any way, is there any use case when you have an error? If so, please include steps to reproduce.

**Comment by bodedra** [comment]
@pedrobaeza In our case, Some users do not have access to printer rights. They try to print it and got error of access denied.

CC @max3903 @guewen @jgrandguillaume

**Comment by pedrobaeza** [comment]
But using regularly?

File: base_report_to_printer/views/printing_printer.xml [hunk 1]
```diff
@@ -4,7 +4,7 @@
     <menuitem name="Printing"
         id="printing_menu" sequence="1"
         parent="base.menu_administration"
-        groups="printing_group_manager"
+        groups="printing_group_manager,base.group_user"
```
Comment (max3903) [inline base_report_to_printer/views/printing_printer.xml]: @bodedra I don't think `group_user` can access the Settings app...

**Comment by max3903** [comment]
@bodedra What is the object in the access denied message?

**Review by max3903 — APPROVED** [review]

**Review by yvaucher — APPROVED** [review]
LGTM, It follows the same logic than base modules

**Pull request merged**